libs/ui-lib/lib/cim/components/ClusterDeployment/ClusterDetailsFormFields.tsx

```diff
@@ -54,10 +60,27 @@ export const ClusterDetailsFormFields: React.FC<ClusterDetailsFormFieldsProps> =
   isNutanix,
   cpuArchitectures,
   allowHighlyAvailable,
+  agentClusterInstall,
 }) => {
+  const { t } = useTranslation();
   const { values } = useFormikContext<ClusterDetailsValues>();
   const { name, baseDnsDomain } = values;
   const [openshiftVersionModalOpen, setOpenshiftVersionModalOpen] = React.useState(false);
+  const { customManifests, isLoading } =
+    CustomManifestService.useCustomManifests(agentClusterInstall);
```

Comment:
`useCustomManifests` should return `K8sResourceCommon[]` so that the consumers do not have to type cast